turbopack/crates/turbopack-core/src/module_graph/chunk_group_info.rs

```diff
@@ -30,7 +30,7 @@ use crate::{
 #[derive(
     Clone, Debug, Default, PartialEq, Serialize, Deserialize, TraceRawVcs, ValueDebugFormat,
 )]
-pub struct RoaringBitmapWrapper(#[turbo_tasks(trace_ignore)] RoaringBitmap);
+pub struct RoaringBitmapWrapper(#[turbo_tasks(trace_ignore)] pub RoaringBitmap);
```

Comment:
You shouldn't need this because of the `Deref`, but doesn't really matter

Reply:
I needed it to construct a new RoaringBitmapWrapper